lms/djangoapps/instructor/views/api.py

```diff
@@ -2133,6 +2133,32 @@ def get_course_forums_usage(request, course_id):
             "for the status of the task. When completed, the report "
             "will be available for download in the table below."
         )
+
+        return JsonResponse({
+            "status": already_running_status
+        })
+
+
+@ensure_csrf_cookie
+@cache_control(no_cache=True, no_store=True, must_revalidate=True)
+@require_level('staff')
+def get_ora2_responses_email(request, course_id):
+    """
+    Pushes a Celery task which will aggregate ora2 responses for a course into a .csv
+    """
+    course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
```

Comment:
Yes, this is something we're all trying to figure out right now. I believe the relevant sections are:

To serialize OpaqueKeys into a string representation, call foo.to_string(), where foo is some type of OpaqueKey object (e.g., either CourseKey, UsageKey, CourseLocator, or BlockUsageLocator).

But SlashSeparatedCourseKey will only work with old style course_ids.

So, I looked for guidance in the upstream master codebase and didn't have to look that far (courseware.views) to find this:

course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)